tests/framework/filters/HostControlTest.php

```diff
@@ -0,0 +1,104 @@
+<?php
+
+namespace yiiunit\framework\filters;
+
+use Yii;
+use yii\base\Action;
+use yii\base\ExitException;
+use yii\filters\HostControl;
+use yii\web\Controller;
+use yiiunit\TestCase;
+
+class HostControlTest extends TestCase
+{
+    protected function setUp()
+    {
+        parent::setUp();
+
+        $_SERVER['SCRIPT_FILENAME'] = "/index.php";
+        $_SERVER['SCRIPT_NAME'] = "/index.php";
+
+        $this->mockWebApplication();
+    }
+
+    /**
+     * @return array test data.
+     */
+    public function dataProviderFilter()
+    {
```

Comment:
shouldnt it be filterDataProvider, consistent with PhpUnits examples https://phpunit.de/manual/current/en/writing-tests-for-phpunit.html#writing-tests-for-phpunit.data-providers in which a format is used with the adjective before dataProvider?